nautobot_netbox_importer/generator/source.py

```diff
@@ -443,10 +457,15 @@ def get_summary(self, content_type_id) -> ModelSummary:
     def set_identifiers(self, identifiers: Iterable[FieldName]) -> None:
         """Set identifiers for the model."""
         if self.identifiers:
-            raise ValueError(f"Duplicate identifiers {identifiers} for {self.identifiers}")
+            if list(identifiers) != self.identifiers:
```

Comment:
should these be sorted before comparison?

Reply:
Good idea, added sorting.